### ISSUE TITLE
Application is using many old dependencies that need to be updated

### ISSUE THREAD

**Issue by ghost** [DemocracyLab/CivicTechExchange#1030]
When viewing the dependencies of the project, both backend dependencies (`requirements.txt`) and frontend dependencies (`package.json`) I see many packages are still using very old versions, examples:
`gunicorn`: current version 19.6, latest version is 21.2
`redis`: current version 3.5.3, latest version is 5.0.0
`webpack`: current version is 4, latest version is 5
`flow-bin`: current version is 0.75, latest version is 0.215.1
and many others

I plan to contribute to this issue by updating packages one-by-one in separate PRs, that is each PR will upgrade one dependency whether backend or frontend.

We need to upgrade these packages:
- [ ] #1031
- [ ] dj-database-url
- [ ] better to pin Django to 4.2.4
- [ ] django-allauth
- [ ] django-appconf
- [ ] django-compressor
- [ ] django-cors-headers
- [ ] django-debreach
- [ ] django-registration
- [ ] django-rq
- [ ] django-s3direct
- [ ] django-storages
- [ ] django-timezone-field
- [ ] docutils
- [ ] gunicorn
- [ ] jmespath
- [ ] psutil
- [ ] psycopg2 to psycopg 3
- [ ] rcssmin
- [ ] redis
- [ ] requests
- [ ] rq
- [ ] simplejson
- [ ] six
- [ ] whitenoise
And frontend packages:
- [ ] babel packages
- [ ] @svgr/webpack
- [ ] @tryghost/content-api
- [ ] aws-sdk migration to modularized v3
- [ ] remove bluebird
- [ ] core-js 
- [ ] css-loader
- [ ] flow-bin
- [ ] jest dependencies
- [ ] webpack and its plugins
- [ ] prettier
- [ ] react to v18
- [ ] react-bootstrap
- [ ] react-datepicker
- [ ] react-file-drop 
- [ ] react-helmet
- [ ] react-google-recaptcha
- [ ] react-image-crop
- [ ] react-markdown
- [ ] react-select
- [ ] react-sortablejs
- [ ] regenerator-runtime
- [ ] repeating

**Comment by abnud1** [comment]
@PeterBreen Yes I know, I've already dealt with major upgrades before and I know how to do it again

**Comment by ddfridley** [comment]
@abnud1 Hi, I'm helping with development focus for DemocarcyLab. We've created a [DemocracyLab Dashboard](https://github.com/orgs/DemocracyLab/projects/1/views/1?layout=board) and would like to update the status of this issue.  

I know that you have one PR posted, that we have in the In Review column. But what is the status of the rest of this.  I know that QA testing is a bottleneck.  Is it possible to move forward on the other items before that one goes through QA. 

Are you available to continue working on this? If not please let me know or unassign yourself. 

Thanks!
